From bb05a4fdcec75b4f9f6ec24996a156a9b38e29a2 Mon Sep 17 00:00:00 2001
From: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
Date: Fri, 19 Feb 2016 09:37:20 -0600
Subject: [PATCH] enable combine libs

This in order to build all DPDK libraries into a single
library.

Signed-off-by: Mario Carrillo <mario.alfredo.c.arevalo@intel.com>
---
 config/common_linuxapp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common_linuxapp b/config/common_linuxapp
index 74bc515..daa9c74 100644
--- a/config/common_linuxapp
+++ b/config/common_linuxapp
@@ -86,7 +86,7 @@ CONFIG_RTE_BUILD_SHARED_LIB=n
 #
 # Combine to one single library
 #
-CONFIG_RTE_BUILD_COMBINE_LIBS=n
+CONFIG_RTE_BUILD_COMBINE_LIBS=y
 
 #
 # Use newest code breaking previous ABI
-- 
2.5.0

